style.css

```diff
@@ -0,0 +1,550 @@
+  }
+
+  .container__article-bali {
+    padding-left: 40px;
+  }
+}
+
+@media screen and (max-width: 768px) {
+
+  .picture {
+    display: none;
+  }
+
+  .bali__btn {
+    margin: 0 auto;
+  }
+}
+
+@media screen and (max-width: 560px) {
+
+  .header.active::after {
+    top: 0;
+  }
+
+  .header {
+    padding: 30px 20px 0;
+    margin: 0 0 120px 0;
+  }
+
+  .logo {
+    height: 30px;
+    width: auto;
+  }
+
+  .nav {
+    display: none;
+    transition: .6s ease;
+  }
+
+  .nav__toggle {
+    display: block;
+  }
+
+  .picture {
+    display: none;
+  }
+
+  .container {
+    padding: 0;
+  }
+
+  .bali {
+    align-items: center;
+    padding: 0;
+  }
+
+  .bali__text {
+    font-size: 18px;
+    align-items: center;
+    padding-top: 30px;
+  }
+
+  .nav__burder {
+    display: block;
+    width: 25px;
+    height: 17px;
+    position: relative;
+    z-index: 10;
+    right: 0;
+  }
+
+  .nav__burder::before,
+  .nav__burder::after {
+    content: "";
+    background-color: #fff;
+    width: 100%;
+    height: 3px;
+    position: absolute;
+    transition: .3s ease;
+  }
+
+  .nav__burder::before {
+    top: 0;
+  }
+
+  .nav__burder::after {
+    bottom: 0;
+  }
+
+  .burger_line {
+    position: absolute;
+    width: 100%;
+    height: 3px;
+    background-color: #fff;
+    top: 7px;
+  }
+
+  .burger_line.active {
+    display: none;
+  }
+
+  .nav.active {
+    width: 100vw;
+    height: 100vh;
+    text-align: center;
+    left: 0;
+    top: 0;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    transition: .6s ease;
+  }
+
+  .nav__burder.active::after,
+  .nav__burder.active::before {
+    background-color: #000;
+  }
+
+  .nav__burder.active::before {
+    transform: rotate(45deg);
+    top: 4px;
+  }
+
+  .nav__burder.active::after {
+    transform: rotate(-45deg);
+    bottom: 10px;
+  }
+
+  .nav.active a {
+    color: #000;
+    animation: opacity 1s ease;
+    margin-bottom: 50px;
+    font-size: 30px;
+    font-weight: bold;
+  }
+
+  @keyframes opacity {
+
+    0% {
+      opacity: 0;
+    }
+
+    100% {
+      opacity: 1;
+    }
+  }
+}
+
+@media screen and (max-width: 490px) {
+
+  .header {
+    padding: 41px 30px 0 42px;
+    margin: 0 0 90px 0;
+  }
+}
+
+@media screen and (max-width: 375px) {
+
+  .logo {
+    width: 104px;
+    height: 39px;
+  }
+
+  .header {
+    margin: 0 0 55px 0;
+  }
+
+  .container__article-bali {
+    margin: 0;
+  }
+
+  .bali__subtitle {
+    margin: 0 0 15px 0;
+    top: 151px;
+  }
+
+  .bali__text {
+    margin: 0 39px 33px 31px;
+    font-size: 18px;
+    line-height: 22px;
+    text-align: center;
+  }
+
+  .bali-wrapper {
+    right: 11px;
+  }
+
+  .body {
+    background: url(pic/backgroundmobile.png);
+    background-repeat: no-repeat;
+  }
+}
+
+@media screen and (max-width: 360px) {
+
+  .bali__title {
+    font-size: 150px;
+    line-height: 200px;
+  }
+
+  .bali__subtitle {
+    font-size: 50px;
+    line-height: 70px;
+    top: 130px;
+  }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Indonesia/pullRequest?prid=5674203)